shared/util/saga.js

```diff
@@ -99,36 +107,6 @@ function safeTakeLatest (pattern: string | Array<any> | Function | Channel<any>,
   return safeTakeLatestWithCatch(pattern, () => {}, worker, ...args)
 }
 
-// take on pattern. If pattern happens while the original one is running just ignore it
```

Comment:
I entirely removed this cause i think its overly complicated and likely unnecessary. If you really want this behavior you can do something like this (https://github.com/redux-saga/redux-saga/issues/589#issuecomment-259346425). Originally i was going to change this to use that but the redux takeLatest/takeEvery actually does a bunch of shit to make it so you can call yield takeEvery().  Our usage of yield [a,b,c] actually hid the fact that if you do
yield safeTakeSerially(a) it actually stops the thread from continuing.

For example  

yield takeEvery(...)
yield takeLatest(...)
yield takeEvery(...)
console.log('i see this')

will all execute and they'll all be forked correctly.

yield safeTakeSerially(...)
console.log('i can't see this')

will not console log correctly cause this thing actually returns a task instead of an iterator.

Given this i just removed this cause we only use this in one place and its kinda crappy.
if you do
yield * safeTakeSerially it will work but thats also very confusing why its different than the other helpers